powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/v3/PowerAuthServiceImpl.java

```diff
@@ -928,4 +928,153 @@ public CommitUpgradeResponse commitUpgrade(CommitUpgradeRequest request) throws
         }
     }
 
+    @Override
+    @Transactional
+    public CreateRecoveryCodeResponse createRecoveryCode(CreateRecoveryCodeRequest request) throws Exception {
+        if (request.getApplicationId() <= 0L || request.getUserId() == null || request.getPukCount() < 1 || request.getPukCount() > 100) {
```

Comment:
Let's store the value 100 in a constant here in the class so that we can change the number easier.

Reply:
Yes, as it should be.

Reply:
Fixed.